services/port_manager/src/main/java/com/futurewei/alcor/portmanager/util/GoalStateUtil.java

```diff
@@ -270,9 +273,9 @@ private static int protocolToNumber(String protocol) {
             } else if (entity instanceof SubnetEntity) {
                 SubnetEntity subnetEntity = (SubnetEntity) entity;
                 subnetEntityMap.put(subnetEntity.getId(), subnetEntity);
-            } else if (entity instanceof SecurityGroupEntity) {
-                SecurityGroupEntity securityGroupEntity = (SecurityGroupEntity) entity;
-                securityGroupEntityMap.put(securityGroupEntity.getId(), securityGroupEntity);
+            } else if (entity instanceof SecurityGroup) {
+                SecurityGroup securityGroupEntity = (SecurityGroup) entity;
```

Comment:
Seems that the latest Alcor/master is not merge yet. We should have SecurityGroupEntity class here @chenpiaoping

Reply:
I renamed it SecurityGroup again.

Reply:
Okay it is fine.